README.md

```diff
@@ -83,43 +47,44 @@ To use the Material theme, import the correspondent file from the `theme/materia
 
 - The component with the Lumo theme:
 
-  `theme/lumo/vaadin-details.html`
+  `theme/lumo/vaadin-details.js`
 
 - The component with the Material theme:
 
-  `theme/material/vaadin-details.html`
+  `theme/material/vaadin-details.js`
 
-- Alias for `theme/lumo/vaadin-details.html`:
+- Alias for `theme/lumo/vaadin-details.js`:
 
-  `vaadin-details.html`
+  `vaadin-details.js`
 
-
-## Running demos and tests in a browser
+## Running API docs and tests in a browser
 
 1. Fork the `vaadin-details` repository and clone it locally.
 
-1. Make sure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
+1. Make sure you have [node.js](https://nodejs.org/) 12.x installed.
 
-1. When in the `vaadin-details` directory, run `npm install` and then `bower install` to install dependencies.
+1. Make sure you have [npm](https://www.npmjs.com/) installed.
 
-1. Make sure you have [polymer-cli](https://www.npmjs.com/package/polymer-cli) installed globally: `npm i -g polymer-cli`.
+1. When in the `vaadin-details` directory, run `npm install` to install dependencies.
 
 1. Run `npm start`, browser will automatically open the component API documentation.
 
-1. You can also open demo or in-browser tests by adding **demo** or **test** to the URL, for example:
+1. You can also open visual tests, for example:
 
-  - http://127.0.0.1:3000/components/vaadin-details/demo
-  - http://127.0.0.1:3000/components/vaadin-details/test
+  - http://localhost:8000/test/visual/default.html
```

Comment:
There is no `default.html`, but that's a nit can be changed later on

Reply:
Thanks, fixed.